netmanager/src/views/layouts/common/Sidebar/components/NetworkDropdown.js

```diff
@@ -11,110 +19,101 @@ import {
 } from 'redux/AccessControl/operations';
 import { loadDevicesData } from 'redux/DeviceRegistry/operations';
 import { loadSitesData } from 'redux/SiteRegistry/operations';
-import { ArrowDropDown } from '@material-ui/icons';
 
-const StyledMenu = withStyles({
-  paper: {
-    border: '1px solid #d3d4d5',
-    width: '170px'
-  }
-})((props) => (
-  <Menu
-    elevation={0}
-    getContentAnchorEl={null}
-    anchorOrigin={{
-      vertical: 'bottom',
-      horizontal: 'center'
-    }}
-    transformOrigin={{
-      vertical: 'top',
-      horizontal: 'center'
-    }}
-    {...props}
-  />
-));
-
-const StyledMenuItem = withStyles((theme) => ({
-  root: {
-    '&:focus': {
-      backgroundColor: theme.palette.primary.main,
-      '& .MuiListItemIcon-root, & .MuiListItemText-primary': {
-        color: theme.palette.common.white
-      }
-    }
+const useStyles = makeStyles((theme) => ({
+  listItem: {
+    '&:hover': {
+      backgroundColor: theme.palette.grey[200],
+    },
+    fontWeight: '1rem',
+    color: '#175df5'
+  },
+  listItemText: {
+    color: '#175df5'
+  },
+  selectedListItem: {
+    backgroundColor: 'green',
+  },
+  activeListItem: {
+    backgroundColor: 'white',
   }
-}))(MenuItem);
+}));
 
 export default function NetworkDropdown({ userNetworks }) {
   const dispatch = useDispatch();
-  const [anchorEl, setAnchorEl] = useState(null);
   const [selectedNetwork, setSelectedNetwork] = useState(null);
+  const [openDialog, setOpenDialog] = useState(false);
+  const classes = useStyles();
 
   useEffect(() => {
     const activeNetwork = JSON.parse(localStorage.getItem('activeNetwork'));
     if (activeNetwork) {
       setSelectedNetwork(activeNetwork);
-    } else {
+    } else if (userNetworks.length > 0) {
       setSelectedNetwork(userNetworks[0]);
```

Comment:
Why the extra else if? `if(userNetworks.length > 0){` @danielmarv

Reply:
i have resolved the requested changes